README.md

```diff
@@ -59,17 +59,48 @@ So, in summary, here's what we need to do:
 
 ## IV. Implementation
 
-Implement wall following to make the car drive autonomously around the Levine Hall map. Follow the inner walls of Levine. Which means follow left if the car is going counter-clockwise in the loop. (The first race we run will be counter-clockwise). You can implement this node in either C++ or Python.
+Implement wall following to make the car drive autonomously around the Levine Hall map. Follow the inner walls of Levine. Which means follow left- make sure the the car is going counter-clockwise in the loop. (The first race we run will be counter-clockwise). You can implement this node in either C++ or Python.
 
 ## V. Deliverables and Submission
 
-**Deliverable 1**: After you're finished, update the entire skeleton package directory with your `wall_follow` package and directly commit and push to the repo Github classroom created for you. Your commited code should start and run in simulation smoothly.
+**Deliverable 1**: After you're finished, update the entire skeleton package directory with your team's `wall_follow` package and directly commit and publish to GitHub. You will need to make sure the TA is added as a collaborator. Your committed code should start and run in simulation smoothly.The package should include a launch file that allows students to change variables from the command line for the following:
+- P, I, and D tuning variables
```

Comment:
Please provide the command line arguments with an example so that it's consistent across submissions.